source/projects/webRadio/de/index.md

```diff
@@ -0,0 +1,136 @@
+
+Zum Verbinden mit dem Radio sind folgenden Befehle notwendig (wobei die angegebene MAC-Adresse des Radios durch die des eigenen Radios ausgetauscht werden muss):
+```shell
+$ bluetoothctl
+Agent registered
+
+$ [bluetooth]# power on
+changing power on succeeded
+
+$ [bluetooth]# default-agent
+Default agent request successful
+
+$ [bluetooth]# pairable on
+changing pairable on succeeded
+
+//wichtiger command, falls man sich erneut(!) mit dem gleichen Gerät verbinden möchte
+$ [bluetooth]# remove 12:34:56:78:9A:BC
+changing pairable on succeeded
+
+// Umgebung nach Bluetooth-Devices scannen; MAC-Adresse des Radios sollte erscheinen
+$ [bluetooth]# scan on
+Discovery started
+[CHG] Device 12:34:56:78:9A:BC 12-34-56-78-9A-BC
+
+$ [bluetooth]# scan off
+Discovery stopped
+
+//mit dem Radio pairen (!= verbinden)
+$ [bluetooth]# pair 12:34:56:78:9A:BC
+Attempting to pair with 12:34:56:78:9A:BC
+[CHG] Device 12:34:56:78:9A:BC Connected: yes
+[CHG] Device 12:34:56:78:9A:BC ServicesResolved: yes
+[CHG] Device 12:34:56:78:9A:BC Paired: yes
+Pairing successful
+
+//überprüfen ob das Pairing wirklich geklappt hat; Radio sollte gelistet werden
+$ [bluetooth]# devices
+Device 12:34:56:78:9A:BC MeinRadioName
+
+//automatisch mit dem Gerät (Radio) verbinden, wenn es eingeschaltet wird
+$ [bluetooth]# trust 12:34:56:78:9A:BC
+[CHG] Device 12:34:56:78:9A:BC Trusted: yes
+Changing 12:34:56:78:9A:BC trust succeeded
+
+//Finally: mit dem Radio verbinden
+$ [bluetooth]# connect 12:34:56:78:9A:BC
+```
+
+## Web Radio per Autostart abspielen
+
+Damit der Aha-Effekt größer und das Ergebnis hörbar ist, haben wir das Shell-Skript um einen weiteren Command ergänzt. Mit dem Command
+```shell
+$ vlc meinePlaylist.m3u
+```
+öffnet der bereits vorinstalliert VLC-Mediaplayer die eigens erstellte Playlist "meinePlaylist". Alle darin befindlichen Dateien wie mp3-Dateien oder weitere Playlists, beispielsweise die Playlist eures Lieblings-Web-Radios, werden mit dem Command nacheinander abgespielt. In die Playlist muss demnach lediglich eine URL zu entsprechenden Web-Radiosender hinterlegt werden.
+
+**Last but not least: das Shell-Skript in den Autostart einbinden.** Um ein Programm in den Autostart des Raspberry's einzubinden gibt es verschiedene Möglichkeiten. In unserem Tutorial haben wir eine **.dektop-Datei** erstellt, um unser Skript mit jedem Hochfahren des RasPi's auszuführen. 
+```shell
+$ sudo geany /etc/xdg/autostart/NameDerDatei.desktop&
+```
+Nice To Know: durch das kaufmännische "Und" am Ende des Befehls, könnt ihr das Terminal weiterhin bedienen, obwohl der Texteditor geöffnet ist.
+Die .desktop-Dateien haben eine vorgeschriebene schematische Struktur, die es zu berücksichtigen gilt.
+```
+[Desktop Entry]
+Type=Application
+Name=irgendeinName
+Terminal=false
+Exec=sh /usr/bin/meinSkript.sh
```

Comment:
Better to use `/bin/sh` then only `sh` you never now when the file is loaded and if `bin` is already in `$PATH`

Reply:
nice to now --> changed to `/bin/sh`